assignment1.txt

```diff
@@ -0,0 +1,6 @@
+Mariposa Lee
+1. A compiler helps the computer understand code by processing a piece of code written in a specific language to another language that the computer can understand. In this case, we program the code first in C. Then, the compiler takes the code in C and converts it into instructions readable by the computer. 

```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.